shapely/geometry/geo.py

```diff
@@ -1,7 +1,7 @@
 """
 Geometry factories based on the geo interface
 """
-
+import numpy as np
```

Comment:
@koshy1123 numpy is still an optional dependency of shapely, so we need to work on adding support for arrays in a way that doesn't import numpy here.

Reply:
Ah, got it - sure! I'll take your suggestion from the review

Reply:
I went with an import check instead. Does that work? PTAL: https://github.com/Toblerity/Shapely/pull/852/commits/bc3b0a72646e2cfd763db3061240aaa16b219c6d

Reply:
Also, makes me think the Travis CI builds for `NUMPY=0` aren't working correctly? They passed for my previous commit.

Reply:
I'm okay with the import check. It's possible that there's been a regression in testing with numpy, I haven't looked closely at that in a while.